requirements.txt

```diff
@@ -7,4 +7,9 @@ pymongo==2.8
 Flask==0.10.1
```

Comment:
ok, it works, but then docker become an extra requirement for development. It sounds really weird. It expect extra work to integrate it with debugging tools. I'm not sure it is a good way to go.